README.md

```diff
@@ -1,5 +1,9 @@
 # Ethereum Bloom Filter
 
+[![Build Status](https://travis-ci.org/ethereum/eth-bloom.svg?branch=master)](https://travis-ci.org/ethereum/eth-bloom)
+[![PyPI version](https://badge.fury.io/py/eth-bloom.svg)](https://badge.fury.io/py/eth-bloom)
+[![Python versions](https://img.shields.io/pypi/pyversions/eth-bloom.svg)](https://pypi.python.org/pypi/eth-bloom)
```

Comment:
Thanks for collecting these. Maybe add it in a different PR, since this one will probably be closed.